src/query/regex_flag.rs

```diff
@@ -0,0 +1,14 @@
+//! Regex Bit Flags
+/// Used to change the Regex Mode in Filters
+pub enum RegexFlag {
```

Comment:
Why is this under the `query` module and not the `exp` module? These flags are only used in filter expressions now, right?

Reply:
It already was there before. I didnt move it to not break compatibility anywhere.

Reply:
That does not seem to be the case. This module does not exist in master branch, but is being added only in this PR. Also, it gets re-exported as `aerospike::RegexFlag`, so we should be able to move it around internally without breaking any client code. (And also, also, we are breaking backward compatibility in the v1.0 release anyway. :-))